### PR TITLE
Release 0.5.3

### PR DESCRIPTION
**Pull request by mark-prins**
Updates master from develop ahead of a v0.5.3 release.

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -4,8 +4,8 @@ buildscript {
     ext {
         buildToolsVersion = "30.0.2"
         minSdkVersion = 21
-        compileSdkVersion = 30
-        targetSdkVersion = 30
+        compileSdkVersion = 31
```
Comment (andreievg) [inline android/build.gradle:7]: As per comment in telegram, we need to add some permissions: https://developer.android.com/guide/topics/connectivity/bluetooth/permissions#declare

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -8,26 +8,11 @@
...
-      android:name=".MainActivity"
-      android:label="@string/app_name"
-      android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
-      android:launchMode="singleTask"
-      android:theme="@style/Theme.App.SplashScreen"
-    >
+  <uses-permission android:name="android.permission.RECORD_AUDIO"/>
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+  <uses-feature android:name="android.hardware.bluetooth_le" android:required="true"/>
+  <application android:name=".MainApplication" android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher" android:allowBackup="false" android:theme="@style/AppTheme" android:usesCleartextTraffic="true" android:requestLegacyExternalStorage="true">
```
[the remark this answers is not in the document]
Comment (mark-prins) [reply android/app/src/main/AndroidManifest.xml:14]: Yeah, the formatting is annoying me. I did already re-save to try and recover the original formatting, but no change.
The only change was for `android:exported` but I think I can un-bump the SdkVersion, so will just revert the whole file

[the remark this answers is not in the document]
Comment (mark-prins) [reply android/app/src/main/AndroidManifest.xml:14]: Turns out the original formatting is the incorrect version - at least according to me. I've reverted the change and vscode formats as per.

**Review by andreievg — APPROVED** [review]
Thanks @mark-prins, looks good.

I tried making changes and re saving manifest, formatting looks good.

**Comment by chetstone** [comment]
Looks good, Mark, at least for Blue Maestro.



I've been playing with this release for a while. Installed it with a clean install on June 9 and added 2 Blue Maestro and 1 BT510 sensors. It worked for a couple of days then I stopped using it. (I occasionally use the tablet for other things.) This week the summer heat started and I started monitoring temps closely. (I use it to keep tabs on which windows I should open/close to keep my house cool.) 



When I started using it again, all the intermediate data downloaded OK from the Blue Maestro sensors except for just 1 or 2 readings. (I have it set to take a reading every five minutes.) 



However, on the BT510, 11 days of data are missing. 



Since I started using it regularly again, all 3 sensors (and a fourth Blue Maestro I added a few days ago) are reporting consistently and I'm no longer having any problems with sensors disconnecting and having to be re-paired. Even after 11 days, they all came back online with no problems.



Cheers

Chester






---- On Mon, 05 Jun 2023 19:40:53 -0600 Mark Prins ***@***.***> wrote ---





Merged https://github.com/openmsupply/msupply-cold-chain/pull/239 into master.

**Comment by mark-prins** [comment]
Hi Chester! Great to hear from you - and thanks for the feedback. 
We've had a few issues with the Laird sensors, so have focused most of the work recently on the blue maestro devices. They now have a maxi device with a larger battery, which is looking promising as a long lasting alternative.
(I'm on leave currently, which is why the long delay in replying 😉)

**Pull request merged**